components/src/molecules/rating/Rating.spec.js

```diff
@@ -0,0 +1,38 @@
+import { mount } from '@vue/test-utils'
+import ARating from './Rating.vue'
+
+describe('Rating', () => {
+  it('has default structure', () => {
+    const wrapper = mount(ARating)
+
+    expect(wrapper.is('div')).toBe(true)
+    expect(wrapper.classes()).toContain('a-rating')
+    expect(wrapper.classes().length).toBe(1)
+  })
+
+  it('renders slot text when passed', () => {
+    const wrapper = mount(ARating, {
+      slots: {
+        item: `<span data-test="item">x</span>`
+      }
+    })
+
+    expect(wrapper.find('.a-rating > button > span').exists()).toBe(true)
```

Comment:
hmm, you don't check slot text but only if span tag exists, use `text` or `html` method